cc/src/test/scala/me/sgrouples/rogue/cc/ObjectIdSubtypeSpec.scala

```diff
@@ -1,4 +1,4 @@
-package me.sgrouples.rogue.cc
+package me.sgrouples.rogue.cc.innertest // TODO why this package name matters
```

Comment:
without the `innertest` package, this file fails to compile with:
```
[error] -- [E008] Not Found Error: /home/sakulk/repos/rogue-fsqio/cc/src/test/scala/me/sgrouples/rogue/cc/ObjectIdSubtypeSpec.scala:48:20
[error] 48 |case class A(_id: A.Id, b: String)
[error]    |                  ^^^^
[error]    |type Id is not a member of object me.sgrouples.rogue.cc.A, but could be made available as an extension method.
[error]    |
[error]    |The following import might make progress towards fixing the problem:
[error]    |
[error]    |  import munit.Clue.generate
[error]    |
[error] -- [E008] Not Found Error: /home/sakulk/repos/rogue-fsqio/cc/src/test/scala/me/sgrouples/rogue/cc/ObjectIdSubtypeSpec.scala:102:19
[error] 102 |case class B(id: B.Id)
[error]     |                 ^^^^
[error]     |type Id is not a member of object me.sgrouples.rogue.cc.B, but could be made available as an extension method.
[error]     |
[error]     |The following import might make progress towards fixing the problem:
[error]     |
[error]     |  import munit.Clue.generate
[error]     |
```
I can't figure out why :man_shrugging: